src/repo/zif_abapgit_repo.intf.abap

```diff
@@ -31,16 +31,18 @@ INTERFACE zif_abapgit_repo
       zcx_abapgit_exception .
   METHODS get_files_remote
     IMPORTING
-      ii_obj_filter   TYPE REF TO zif_abapgit_object_filter OPTIONAL
+      !ii_obj_filter   TYPE REF TO zif_abapgit_object_filter OPTIONAL
+      !iv_ignore_files TYPE abap_bool DEFAULT abap_false
+        PREFERRED PARAMETER ii_obj_filter
```

Comment:
wondering, what is the scenario where we dont want to remove the ignored files? can this parameter be removed? should it be default true?

Reply:
Long term, I can image showing all remote files in AG (like VScode). You could view or even update all of them in AG, too (like readme etc). Then have a view option to hide "ignored files".